docs/metaed-project/Association/UserLMSSectionAssociation.metaed

```diff
@@ -1,37 +1,37 @@
-﻿// SPDX-License-Identifier: Apache-2.0
-// Licensed to the Ed-Fi Alliance under one or more agreements.
-// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-// See the LICENSE and NOTICES files in the project root for more information.
-
```

Comment:
@ggaronzik is it possible to move this license boilerplate back to the top of the file instead of being a footer?

Reply:
MetaEd does not expect any text, even comments, before the initial declaration. So I moved these all to the bottom. I could have them be on the second line but I could build with the comments at the top. @bradbanister may have an alternative suggestion here.

Reply:
@ggaronzik  I'm a bit surprised MetaEd doesn't like it.  Can you open a bug?

Reply:
Somehow this isn't a problem in the main ODS codebase, for instance: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/main/Extensions/Homograph/HomographMetaEd/Association/StudentSchoolAssociation.metaed

Reply:
But we can certainly live with having the text at the bottom of the file for now.

Reply:
Comments no longer a bug - these have been moved to the top of the files again with the latest commit.